provider/puppet/common~copy.yaml

```diff
@@ -16,5 +16,15 @@
 # TODO(nelsonjr): Move all objects out of google/ namespace into google/product/
 'LICENSE': 'templates/LICENSE'
 'Gemfile.lock': 'templates/puppet/Gemfile.lock'
+'lib/google/hash_utils.rb': 'google/hash_utils.rb'
```

Comment:
What are these changes from ~compile to ~copy? If you say it is fine, then, I am ok with it. I just want to make sure you were aware of these.

Reply:
Compiled files have ERB tags and should then have a autogen clause.
Copied files do not and don't need an autogen clause.

This is especially important as some of our libraries are shared between Puppet/Chef/MM. I can't place a ERB tag in that file or it will crash Magic Modules.

In the past, we've been bad about differentiating between compiled files + copied files. This change will make it more important to copy files when they don't have ERB tags.